deps/wazuh_testing/wazuh_testing/scripts/statistical_data_analyzer.py

```diff
@@ -0,0 +1,339 @@
+import argparse
+import os
+import sys
+import pandas as pd
+from prettytable import PrettyTable
+from scipy.stats import ttest_ind, levene, f_oneway
+
+STATS_MAPPING = {
+    'cpu': 'CPU(%)',
+    'memory': 'RSS(KB)',
+    'virtual_memory': 'VMS(KB)',
+    'file_descriptor': 'FD',
+    'read_ops': 'Read_Ops',
+    'write_ops': 'Write_Ops',
+    'disk_read': 'Disk_Read(B)',
+    'disk_written': 'Disk_Written(B)',    
+    'disk_usage': 'Disk(%)',
+    'uss': 'USS(KB)',
+    'pss': 'PSS(KB)',
+    'swap': 'SWAP(KB)',
+}
```

Comment:
The current implementation of the stats mapping units is hardcoded, which limits the module's flexibility with varying data inputs. Additionally, it includes certain default metrics, such as Disk (%), that may not be relevant for future analyses.

I recommend refactoring the module to use classes, which would allow for customizable stats mapping based on the received data. This would also enable the module to ignore metrics that are not provided in the directory, enhancing its adaptability and usefulness.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5622/commits/26dc5cfa8d7f2339d13d5d5820d1ad7cceda3625